src/tests/test_config.py

```
@@ -0,0 +1,42 @@
+"""
+tests/test_config.py — Tests for the production-safety config validator.
+"""
+
+import pytest
+
+from app.config import Settings
+
+
+def _base_kwargs(**overrides):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_base_kwargs`

(ANN202)